### ISSUE TITLE
SyntaxError: Unexpected end of input

### ISSUE THREAD

**Issue by mitel** [AtomLinter/linter-eslint#148]
hi, i get this error with 2.2.0:
SyntaxError: Unexpected end of input
  at Object.parse (native)
  at Object.<anonymous> (/Users/mitel/.atom/packages/linter-eslint/lib/linter-eslint.js:120:28)
  at GeneratorFunctionPrototype.next (native)
  at step (/Users/mitel/.atom/packages/linter-eslint/lib/linter-eslint.js:7:273)

**Comment by iam4x** [comment]
@mitel This error should be catched now, could you open the atom console and copy me the output of ESLint?

**Comment by nocrates** [comment]
Thanks for looking into this, I'm having the same error.  This were great until I upgraded.  I do not know what version I was on before upgrade, but it was only a few weeks old.

Current versions (atom packages):
linter: 1.2.3
linter-eslint: 2.4.1

eslint in /usr/local/bin/eslint, version v0.24.0

This is my console output ( I hope this is what you are looking for):
```
activate `linter-eslint`
index.js:56  Window load time: 1054ms
linter-eslint.js:126  [Linter-ESLint] no output from CLI
linter-eslint.js:127  eslint binary: /usr/local/bin/eslint
linter-eslint.js:128  params: --stdin-filename index.js --config .eslintrc --format /Users/esexton/.atom/packages/linter-eslint/node_modules/eslint-json/json.js --stdin
linter-eslint.js:129  cwd: /Users/esexton/Dropbox/src/attachmentbot
```

When I run eslint ( /usr/local/bin/eslint ) from command-line, I do receive output.
```
$ cat index.js | eslint --stdin-filename index.js --config .eslintrc --format /Users/esexton/.atom/packages/linter-eslint/node_modules/eslint-json/json.js --stdin
{"results":[{"filePath":"index.js","messages":[{"ruleId":"quotes","severity":2,"message":"Strings must use singlequote.","line":1,"column":0,"nodeType":"Literal","source":"\"use strict\";"},{"ruleId":"no-unused-vars","severity":2,"message":"
...
( etc. )

```

Thank you!

**Comment by nocrates** [comment]
Good news!

After uninstalling linter-eslint and reinstalling, it works!
So, apparently the problem was with the upgrade from old to new.

**Comment by mitel** [comment]
here you go with 2.4.1 and eslint 0.24.1:

cwd: /Users/mitel/develop/react-stuff/flux-stuff/redux-example
linter-eslint.js:126 [Linter-ESLint] no output from CLI
linter-eslint.js:127 eslint binary: /Users/mitel/.node/bin/eslint
linter-eslint.js:128 params: --stdin-filename src/js/components/HomePage.jsx --config .eslintrc --format /Users/mitel/.atom/packages/linter-eslint/node_modules/eslint-json/json.js --stdin
linter-eslint.js:129 cwd: /Users/mitel/develop/react-stuff/flux-stuff/redux-example
events.js:141 Uncaught Error: write EPIPE
linter-eslint.js:126 [Linter-ESLint] no output from CLI
linter-eslint.js:127 eslint binary: /Users/mitel/.node/bin/eslint
linter-eslint.js:128 params: --stdin-filename src/js/components/HomePage.jsx --config .eslintrc --format /Users/mitel/.atom/packages/linter-eslint/node_modules/eslint-json/json.js --stdin
linter-eslint.js:129 cwd: /Users/mitel/develop/react-stuff/flux-stuff/redux-example

**Comment by AVVS** [comment]
Same, however, works if passed directly. eslint 2.4.1
Previous version dont work either

```
ctivate `linter-eslint`
index.js:56 Window load time: 1058ms
/Users/vitaly/.atom/packages/linter-eslint/lib/linter-eslint.js:136 [Linter-ESLint] no output from CLI
/Users/vitaly/.atom/packages/linter-eslint/lib/linter-eslint.js:137 eslint binary: /Users/vitaly/.nvm/versions/io.js/v2.2.1/bin/eslint
/Users/vitaly/.atom/packages/linter-eslint/lib/linter-eslint.js:138 params: --stdin-filename src/configs/default.js --config .eslintrc --format /Users/vitaly/.atom/packages/linter-eslint/node_modules/eslint-json/json.js --stdin
/Users/vitaly/.atom/packages/linter-eslint/lib/linter-eslint.js:139 cwd: /Users/vitaly/projects/arkmail-backend

Vitalys-MacBook-Pro:arkmail-backend vitaly$ cat src/configs/default.js | eslint --stdin-filename src/configs/default.js --config .eslintrc --format /Users/vitaly/.atom/packages/linter-eslint/node_modules/eslint-json/json.js --stdin
{"results":[{"filePath":"src/configs/default.js","messages":[],"errorCount":0,"warningCount":0}]}
```

**Comment by iam4x** [comment]
We have an issue with $PATH env not set correctly when Atom isn't ran from CLI. Let's keep the discussion in one place #149
